hack/run-locally.sh

```diff
@@ -94,11 +101,64 @@ function wait_for_cluster() {
     fi
 }
 
+function from_scratch() {
+    if [[ -n "${INSTALLER_DIR:-}" ]]; then
```

Comment:
It seems to me that you should be able to decide if this is "from scratch" based on the contents of `CLUSTER_DIR`...

Reply:
I picked installer_dir because (a) we need it to do from_scratch, and (b) you don't need it to do the not-from-scratch. You need cluster-dir either way.

Reply:
@danwinship your suggestion is better, fixing that up.